x-pack/plugins/reporting/server/routes/internal/diagnostic/browser.ts

```diff
@@ -54,7 +54,7 @@ export const registerDiagnoseBrowser = (reporting: ReportingCore, logger: Logger
       const logsToHelpMap = logsToHelpMapFactory(docLinks);
       try {
         const { screenshotting } = await reporting.getPluginStartDeps();
-        const logs = await lastValueFrom(screenshotting.diagnose());
+        const logs = await lastValueFrom(screenshotting!.diagnose());
```

Comment:
Instead of force casting, wdyt about explicitly throwing in these cases? Something like:

```suggestion
        if (!screenshotting) throw new Error('Screenshotting is not enabled.');
        const logs = await lastValueFrom(screenshotting.diagnose());
```

Reply:
I like this so much better thanks!!!